src/main/java/domain/AttendanceAlertLevel.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+public enum AttendanceAlertLevel {
+    DISMISSED(6, "제적"),
+    COUNSEL_REQUIRED(3, "면담"),
+    CAUTION(2, "경고"),
+    NORMAL(0, "일반"),
+    ;
+
+    int absenceLimit;
+    String name;
```

Comment:
```suggestion
    final int absenceLimit;
    final String name;
```

`final` 키워드는 왜 필요할까요?

Reply:
네, enum에서 필드에 final 키워드가 필요한 이유는

- enum은 본질적으로 불변의 성격(상수 개념)을 가지기 때문에 내부 필드가 변경된다면 상수로서의 의미가 퇴색
- enum은 인스턴스를 생성할 수 없고 열거 인자들이 public static final로 정의 되어 thread간 공유가 가능해 필드를 final로 선언하지 않으면 개발자가 의도치 않게 변경된 값을 사용할 위험이 있어 final 키워드로 컴파일 타임에 이를 방지

등의 이유가 있습니다.
따라서 필드에 final 키워드를 선언해주어 enum의 설계 의도를 명확하게 하고 안정성을 지켜야합니다.

이런 본질적인 의문을 가지지 않고 큰 생각 없이 final을 사용했던 것 같습니다..일깨워주셔서 감사합니다.

Reply:
꼼꼼히 찾아보셨네요!! 너무 좋습니다! 💯